src/app/components/tabs/index.html

```diff
@@ -1,15 +1,292 @@
+				</div>
+				<div class="screenshot screenshot-danger">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-double-tab-vert.png" />
+					<div class="caption sky-text-danger">Don't combine vertical tabs with horizontal tabs.</div>
+				</div>
+			</div>
+		</div>
+    
+    <div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<p>The tabs are likely to overflow the available horizontal space, which will lead to the collapsed tab dropdown and a large amount of unused space in the tab bar. Use <a routerLink="/components/vertical-tabs">vertical tabs</a> instead.</p>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot screenshot-danger">
+					<img class="bordered" src="~/assets/img/guidelines/tab/dont-overflow.png" />
+					<div class="caption sky-text-danger">Don't use horizontal tabs when they are likely to overflow.</div>
+				</div>
+			</div>
+		</div>
+		
+	</sky-demo-page-content>
+	
+	<sky-demo-page-content sectionHeading="Anatomy">
+		<div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<div class="anatomy-list">
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">1</div><div class="anatomy-description">Selected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">2</div><div class="anatomy-description">Unselected tab</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">3</div><div class="anatomy-description">Tab border</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">4</div><div class="anatomy-description">Tab count (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">5</div><div class="anatomy-description">Close tab button (optional)</div></div>
+					<div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">6</div><div class="anatomy-description">Open tab button (optional)</div></div>
+				</div>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot">
+					<img class="bordered" srcset="~/assets/img/guidelines/tab/anatomy.png 1x, ~/assets/img/guidelines/tab/anatomy_2x.png 2x" src="~/assets/img/guidelines/tab/anatomy.png" />
+				</div>
+			</div>
+		</div>
+	</sky-demo-page-content>
+	
+	<sky-demo-page-content sectionHeading="Options">
+		<h3>Count</h3>
+			<div class="two-col-right-image sky-margin-stacked-separate">
+				<div class="two-col-text">
+					<p>When you use tabs to hold lists, you can include the list's record count in the tab so users know if there are records to look at without navigating to the tab.</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot">
+						<img class="bordered" src="~/assets/img/guidelines/tab/option-count.png" />
```

Comment:
Should this image have a caption? It seems like maybe it should have something along the lines of "Do display record counts in tabs that contain lists."

Reply:
updating the wording around this guideline. I don't think we want a caption since it's not something that should be done in all cases.

Reply:
OK. Thanks.